templates/templates/component/terraform/fogg.tf.tmpl

```diff
@@ -150,3 +150,11 @@ variable "aws_accounts" {
   {{ end }}
   }
 }
+
+{{- if .LocalsBlock }}
+locals {
+{{ range $key, $val := .LocalsBlock }}
+  {{ $key }} = {{ $val }}
+{{ end }}
```

Comment:
these will always be sorted!

gotemplate sorts map by keys
https://github.com/golang/go/commit/a5950df89e129e93f76aa8391f84beb460ef82be